assignments/assignment-8/assignment-8.2.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment 8.2</title>
+    </head>
+    <body>
+        <h2>Quiz</h2>
+
+        <label for="fullNameInput">Full Name</label>
+        <input
+            type="text"
+            placeholder="Full Name"
+            id="fullNameInput"
+            name="fullName"
+            required
+        /><br />
+        <br />
+        <label for="emailInput">Email</label>
+        <input
+            type="email"
+            placeholder="Email"
+            id="emailInput"
+            name="email"
+            required
+        />
+
+        <h3>Confidence level of answering</h3>
+
+        <label for="confidentLevelInput">Level between 1 and 10</label>
+        <input
+            type="range"
+            id="confidentLevelInput"
+            name="level"
```

Comment:
confidenceLevel is better choice for name attr and ratingInput is better for id and for attr

Reply:
ok thanks